docs/manual/lua-debugging.md

```diff
@@ -16,35 +27,43 @@ The Redis Lua debugger, codename LDB, has the following important features:
 * Pretty printing of Redis and Lua values.
 * Infinite loops and long execution detection, which simulates a breakpoint.
 
-Quick start
----
+## Quick start
 
 A simple way to get started with the Lua debugger is to watch this video
 introduction:
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/IMvRfStaoyM" frameborder="0" allowfullscreen></iframe>
 
-**Important note:** please make sure to avoid debugging Lua scripts using your Redis production server. Use a development server instead. Also note that using the synchronous debugging mode (which is NOT the default) results into the Redis server blocking for all the time the debugging session lasts.
+> Important Note:  please make sure to avoid debugging Lua scripts using your Redis production server.
+Use a development server instead.
+Also note that using the synchronous debugging mode (which is NOT the default) results in the Redis server blocking for all the time the debugging session lasts.
```

Comment:
For new text, we use this style, but please don't reflow old text. It's unnecessary changes.